packages/react-devtools-shared/src/devtools/views/Profiler/ProfilingImportExportButtons.js

```diff
@@ -114,6 +116,7 @@ export default function ProfilingImportExportButtons() {
         onChange={handleFiles}
         tabIndex={-1}
       />
+      <a ref={downloadRef} className={styles.Input} />
```

Comment:
nit: At this point we might consider renaming the `Input` class to e.g. `Hidden` or something but this is minor, given the small scope.

Reply:
Good shout. We can fix this in another PR. I'll reference this comment.